data/data-pipeline/data_pipeline/comparison_tool/README.md

```diff
@@ -0,0 +1,60 @@
+# Score comparisons
+
+## Comparison tool
+
+TODO once the comparison tool has been refactored. 
+
+## Single comparator score comparisons
+
+The goal of this directory is to create interactive 1-to-1 dac list:cejst comparisons. That means that, when this tool is run, you will have comparisons of two true/false classifications. 
+
+This uses `papermill` to parameterize a jupyter notebook, and is meant to be a *lightweight* entry into this analysis. The tool as a whole creates a bunch of comparisons against CEJST data -- but after it runs, you'll have the notebook to re-run and add to if you are so inclined. 
+
+To run:
```

Comment:
Can we add documentation about the purpose and use of the shell script (and instructions to future developers to make sure to add to it)?